Sources/Models/Views/ActivityindicatorView.swift

```diff
@@ -35,6 +35,8 @@ public struct ActivityindicatorView: IBDecodable, ViewProtocol, IBIdentifiable {
     public let variations: [Variation]?
     public let backgroundColor: Color?
     public let tintColor: Color?
+    public let hidden: Bool?
```

Comment:
like the bool we prefer use `is` prefix, so `isHidden`
then to decode we need to add line for name binding (we could be inspired by isMisplaced)

ps: I known `opaque` is not `isOpaque` but I want to change it